### PR TITLE
Added a feedback section and other minor changes

### PR DESCRIPTION
**Pull request by pgomez-redsauce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
-# WDIO with Cucumber project Boilerplate
+# WebdriverIO with Cucumber project Boilerplate
+
+[WebdriverIO](https://webdriver.io/) project boilerplate to get started. Inside this project you will find and example test made on the test blog.
```
Comment (krhoton) [inline README.md:3]:
```suggestion
[WebdriverIO](https://webdriver.io/) project boilerplate to get you started. Inside this project you will find a test example targeting our blog's test environment
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+
+*WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and flakiness test suite.*
 
-[WDIO](https://webdriver.io/) project boilerplate to get started. Inside this project you will find and example test made on the test blog.
 
 ## Pre-requisites
 
-You need Testingbot credentials to be able tu run some parts of this boilerplate which can be found in your [Account](https://testingbot.com/members/user/edit) page.
+You need Testingbot credentials to be able to run some parts of this boilerplate which can be found in your [Account](https://testingbot.com/members/user/edit) page.
```
Comment (krhoton) [inline README.md:10]:
```suggestion
You need Testingbot credentials to be able to run some parts of this boilerplate. They can be found in your [Account](https://testingbot.com/members/user/edit) page.
```

File: README.md [hunk 3]
```diff
@@ -20,39 +23,28 @@ You have to set the environment variables **`TB_KEY`** and **`TB_SECRET`**. Alte
 ## Test running
 
 
-WDIO has different commands to run tests, you can tun a single test, a set of tests or all tests. By default tests are run in a headless manner meaning no browser window will be opened while running the tests and results will be seen in the terminal.
-
-  
+WebdriverIO has different commands to run tests, you can run a single test, a set of tests or all tests. By default tests are run in a headless manner meaning no browser window will be opened while execution and results will be seen in the terminal.
```
Comment (krhoton) [inline README.md:26]:
```suggestion
WebdriverIO has different commands to run tests: you can run a single test, a set of tests or all tests. By default tests are run in headless mode, meaning no browser windows will be opened during execution and results will be seen only in the terminal.
```

File: README.md [hunk 4]
```diff
@@ -20,39 +23,28 @@ You have to set the environment variables **`TB_KEY`** and **`TB_SECRET`**. Alte
...
 To get started, with the following command you will be able to run all your tests.
 
-  
-
 	npm run wdio
 
-  
-
-**Note**: This command is specified in *package.json* file, to run tests with different options, you will have to look at the documentation and modify the command you are using.
+**Note**: This command is specified in *package.json* file, to run tests with different options, you will have to read the documentation and modify the command you are using.
```
Comment (krhoton) [inline README.md:34]:
```suggestion
**Note**: This command is specified in the *package.json* file. To run tests with different options, you will have to read the documentation and modify the command you are using.
```

File: README.md [hunk 5]
```diff
@@ -20,39 +23,28 @@ You have to set the environment variables **`TB_KEY`** and **`TB_SECRET`**. Alte
...
+Inside this folder, you will find components, features and step-definitions.
 
 -  **COMPONENTS**:
-
   
 Each part of the web is divided in components, so you can detail each one without mixing them. Inside here usually you will find a folder for each part (examples are shown in the project).
 
-  
-
 Inside each component, you will have the file for the functions, and also a file where you will write all the selectors/or locators you need to find the elements.
```
Comment (krhoton) [inline README.md:48]:
```suggestion
Inside each component, you will have a file for the functions and a file where you will write all the selectors/locators you need to find the elements.
```

File: README.md [hunk 6]
```diff
@@ -20,39 +23,28 @@ You have to set the environment variables **`TB_KEY`** and **`TB_SECRET`**. Alte
...
 
 **Note**: When code is repeating or you think that you will use it in another function, a component usually named shared is created. There, you will follow the same file organization, but for things that you will share between other components (data, functions, etc).
```
Comment (krhoton) [inline README.md]: I would rewrite this to make it clear that having anything inside "shared" is heavily discouraged and should only be used when unavoidable.

File: README.md [hunk 7]
```diff
@@ -61,17 +53,15 @@ Inside each component, you will have the file for the functions, and also a file
...
 
 -  **STEP-DEFINITIONS**:
 
-This folder is for defining the steps of the features that you create previously. This steps are reusable, it means that if a step is repited in two features, you only need to implement it once.
+This folder is for defining the steps of the features that you create previously. Steps are reusable; if a step is repited in two features, you only need to implement it once.
 
   
 
-**Note**: When code is repeating or you think that you will use it in another function, a component usually named shared is created. There, you will follow the same file organization, but for things that you will share between other components (data, functions, etc).
+**Note**: When code is repeating or you think that you will use it in another function, a component usually named *shared* is created. There, you will follow the same file organization, but for things that you will share between other components (data, functions, etc).
```
Comment (krhoton) [inline README.md]: duplicated?

File: README.md [hunk 8]
```diff
@@ -86,40 +76,41 @@ This folder is for defining the steps of the features that you create previously
...
-	  As mentioned above, we use environment variables to run some tests. For the tests to work on workflows you'll need to configure the environment variables within each git.  
+As mentioned above, we use environment variables to run some tests. For the tests to work on workflows you'll need to configure the environment variables within each git.  
 
-## TESTINGBOT
+## Testingbot device farm
 
-	This boilerplate also do the tests in Testingbot. This is setted in the testingbot configuration file (wdio.testingbot.conf.ts).
-	You can run the tests in testingbot with: 
-			npm run testingbot
+This boilerplate can execute test in the Testingbot device farm. This is setted in the testingbot configuration file (wdio.testingbot.conf.ts).
```
Comment (krhoton) [inline README.md:83]:
```suggestion
This boilerplate can execute tests in the Testingbot device farm. This is set in the testingbot configuration file (wdio.testingbot.conf.ts).
```

File: README.md [hunk 9]
```diff
@@ -86,40 +76,41 @@ This folder is for defining the steps of the features that you create previously
...
+	npx allure open ./allure-report
 
-	**Note**: In the global-setup file we have added a function so that whenever you run 					the tests, the contents of the folder will be removed. If you want to have all the reports to be able to make a comparison, you only have to comment or remove that code as specified.
+**Note**: In the global-setup file we have added a function so that whenever you run the tests, the contents of the folder will be removed. If you want to have all the reports to be able to make a comparison, you only have to comment or remove that code as specified.
 
 
 ## ESLint
 
-This project uses ESLint to statically analyze the code and quickly find problems. All the eslint specification can be found in the *.eslintrc.json* file.
+This project uses ESLint to perform a static code analysis. All the ESLint specification can be found in the *.eslintrc.json* file.
```
Comment (krhoton) [inline README.md:108]:
```suggestion
This project uses ESLint to perform static code analysis. All the ESLint configuration can be found in the *.eslintrc.json* file.
```

File: README.md [hunk 10]
```diff
@@ -129,4 +120,20 @@ Quickly fix problems that were found
 
 	npm run lint:fix
 
-**Note**: This commands can be found inside *package.json*, to run lint with different options you have to look at their official [documentation](https://eslint.org/docs/latest/user-guide/command-line-interface). 
\ No newline at end of file
+**Note**: This commands can be found inside *package.json*. To run lint with different options you should read the official [documentation](https://eslint.org/docs/latest/user-guide/command-line-interface).
```
Comment (krhoton) [inline README.md:118]:
```suggestion
**Note**: These commands can be found inside *package.json*. To run lint with different options you should read the official [documentation](https://eslint.org/docs/latest/user-guide/command-line-interface). 
```

**Review by krhoton — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (pgomez-redsauce) [reply README.md]: True, I am working on that

File: README.md (hunk 7 above)
Comment (pgomez-redsauce) [reply README.md]: Yes, deleted one of the duplicates

**Review by pgomez-redsauce — COMMENTED** [review]
Review done

**Review by krhoton — APPROVED** [review]

**Pull request merged**